### ISSUE TITLE
Tweak on Reward Bag

### ISSUE THREAD

**Issue by RaineAndrews** [Haggle1996/RevolutionPack#408]
TLDR: In my opinion, 8 stacks of HSLA is OP where it is currently falling in the questline

What is the intent for having 8 full stacks of HSLA Steel ingots?  I keep getting this one on restarts and it completely bypasses a lot of the mining I have to do for RotaryCraft.  Its rare, but even as rare as this bag is, 8 stacks of HSLA Steel ingots is very powerful.

How does the reward system work?  Is it tiered so that you can keep crap bags from getting it, or do better bags have better chances of getting it?  In either case, I feel its in too low a tier for the amount that you get.

Imagine: you mined 32 hard ore in 30 seconds, grinded the whole lot in 7 minutes, and smelted the stack of 64 in another  two minutes, then blasted it to HSLA, you would only have a stack of HSLA in 10 minutes.  Even with liberal times, this reward bag gives me nearly 80 minutes of work!

If it were gauged further down the quest path, I would have some ore processing/efficiency upgrades which would dampen the effect of 8 stacks of HSLA!

**Comment by Aivech** [comment]
> 8 stacks of HSLA Steel ingots is very powerful.

A medium farm setup in a pack with no difficulty mods cost me 2-3 stacks of it.

> reward bags

Do you remember the % chance of this particular reward from this bag? If this was a regular reward you could select, that would be OP, but this is one of several possible options.

> 80 minutes of work

The reward bags are supposed to be a random chance to save you a considerable amount of work. I don't remember exactly what the rewards were in Rev|1, but one of them was a full drum of TiCon molten HSLA, or around 2500 HSLA ingots. Guaranteed on quest completion, not as a reward bag.

While yes, that's quite a bit of work you save, overall it's a drop in the oceans of iron Rotarycraft requires. 

TL; DR: Rotarycraft takes a lot of HSLA steel, that's one of several rewards you have a random chance to get, quest rewards in the last pack were even more OP, that's a lot of work you save but in the long run 8 stacks of HSLA is pretty slim pickings from mid-game Rotarycraft on.

**Comment by RaineAndrews** [comment]
4 stacks of HSLA ingots is a 1.5% chance with a basic (lowest tier) bag, and 1.97% chance with Good (the next step up).

8 stacks is less than a percent with basic bags.  Maybe all my early game runs have seen a number of 4 stack drops, because I get multiple HSLA drops before I even have a RotaryCraft blast furnace up and running

**Comment by Kreidos** [comment]
What's the point of rare rewards if they aren't awesome? Besides, you need a lot more than steel to progress through rotarycraft. I got the 8 stack reward in my world and it was super helpful early-on to make basic gearboxes, engines and shafts, but ultimately ran out well before I ever reached the end of the RotaryCraft progression tree. To say it's a punched ticket to breeze through Rotarycraft would be false.

**Comment by Haggle1996** [comment]
8 stacks is not enough to breeze through anything but the early tier of RotaryCraft. Basically, enough to rush fermenters, dc engines, gasoline engines, gearbox, grinder, friction heater, magnetic coil charger, and ac engines -- and that's if you're smart with the resource. If you also make tools and armor, you're still stuck well behind the resource curve.

Uber bags are rare, and most players will not get more than one or two of them. Some of the Lets Players I watch on youtube haven't gotten any, even with 3 people playing in a party (meaning they get 3x the rewards).

All the HSLA bag does is save you mining grind. If it gave you bedrock armor, turbines, or jet fuel, that would be OP -- but all it does is save you a day of pounding hard iron ore and searching for coal.

**Comment by RaineAndrews** [comment]
DaMachinator and Haggle1996 make a good point.  There is a lot of HSLA needed in the life of RotaryCraft!  It might save 80+ minutes, but there are a lot of minutes (lets be honest, tracked in weeks) in RotaryCraft.

**Comment by Haggle1996** [comment]
Regardless, I greatly appreciate suggestions, @RaineAndrews. Please keep them coming.
